docs/reference/manifest/requirements.md

```diff
@@ -1,15 +1,64 @@
+
+1. They are associated with a requirement set.
+2. There is a **Requirements** element in the **VersionOverrides** that specifies that requirement set.
+3. The current platform and Office version combination does not support the requirement set.
+
+Examples of elements in a **VersionOverrides** that provide additional features instead of overriding settings in the base manifest include **WebApplicationInfo** and **EquivalentAddins**.
+
+**Add-in type:** Task pane, Mail
+
+**Valid only in these VersionOverrides schemas**:
+
+- Task pane 1.0
+- Mail 1.0
+- Mail 1.1
+
+For more information, see [Version overrides in the manifest](../../develop/add-in-manifests.md#version-overrides-in-the-manifest).
+
+**Associated with these requirement sets**:
+
+- [AddinCommands 1.1](../requirement-sets/add-in-commands-requirement-sets.md) when the parent **VersionOverrides** is type Taskpane 1.0.
+- [Mailbox 1.3](../../reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md) when the parent **VersionOverrides** is type Mail 1.0.
+- [Mailbox 1.5](../../reference/objectmodel/requirement-set-1.5/outlook-requirement-set-1.5.md) when the parent **VersionOverrides** is type Mail 1.1.
+
+### Remarks
+
+The **Requirements** element serves no purpose in a **VersionOverrides** if it specifies no additional requirements that aren't specfied in a **Requirements** in the base manifest, because if the Office version and platform don't support the requirements in the base manifest, the add-in is not activated and the **VersionOverrides** element is not even parsed. For this reason, you should use a **Requirements** element in a **VersionOverrides** only when both of these conditions are met:
+
+- Your add-in has extra features that are implemented with configuration in a **VersionOverrides** (such as Add-in Commands), and that require a method or requirement set that is *not* specified in a **Requirements** element in the base manifest.
+- Your add-in is useful, and should be activated (but without the extra features), even in a combination of platform and Office version that does not support the requirements needed for the extra features.
+
+> [!TIP]
+> Do not repeat **Requirement**s from the base manifest inside a **VersionOverrides**. Doing so has no effect and is potentially misleading as to the purpose of the **Requirements** element inside a **VersionOverrides**.
```

Comment:
Should this be **Requirements**? (s is outside the bold formatting).

Reply:
```suggestion
> Do not repeat **Requirement** elements from the base manifest inside a **VersionOverrides**. Doing so has no effect and is potentially misleading as to the purpose of the **Requirements** element inside a **VersionOverrides**.
```